stacks-signer/src/client/mod.rs

```diff
@@ -459,7 +456,7 @@ pub(crate) mod tests {
                     &StacksPublicKey::from_slice(ecdsa_public_key.to_bytes().as_slice())
                         .expect("Failed to create stacks public key"),
                 );
-                signer_slot_ids.insert(address, signer_id); // Note in a real world situation, these would not always match
+                signer_slot_ids.push(signer_id); // Note in a real world situation, these would not always match
```

Comment:
Can you update the comment?  Previously it looked like the comment meant that `address` and `signer_id` would not always match.

Reply:
No this comment is accurate. It means that signer id and signer slot id would not always match